README.md

```diff
@@ -1,26 +1,26 @@
-#Security Advisor
+# Security Advisor
 Add-on prototype for warning users when their accounts may have been compromised. Provides
 a warning whenever a user visits a site on the add-on's compromised site list.
-Uses haveibeenpwned.com API to load in compromised site list. Can also use Normandy recipes
+Uses https://haveibeenpwned.com API to load in compromised site list. Can also use Normandy recipes
 to add to list of compromised sites, as well as local data.
 
 # To build an XPI
 ```bash
 $ npm run build
 ```
 
-This will create `@security-advisor-0.0.*.xpi`, which you can install in Firefox.
+This will create `security-advisor.xpi`, which you can install in Firefox.
```

Comment:
It looks like somewhere between **jpm@1.0.7** and latest **jpm@1.2.1** the default naming changed from "@security-advisor-{version}.xpi" to the much saner and predictable "security-advisor.xpi". Updated Circle and README.